pkg/controller/master/supportbundle/manager.go

```diff
@@ -22,7 +22,8 @@ import (
 )
 
 const (
-	HarvesterServiceAccount = "harvester"
+	HarvesterServiceAccount   = "harvester"
+	AdditionalTaintToleration = ":"
```

Comment:
does this mean tolerating all taints? if yes can we either add a comment or change to a proper constant name, thanks.

Reply:
yeah that string `:` gets parsed by sbk into `v1.TolerationOpExists`